internal/pkg/github/githubtesting/archive_reader.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
```

Comment:
Why is this copied into here and not just imported from bufbuild/buf?

Reply:
fixed